### PR TITLE
Update D:\Git\winget-pkgs\manifests\m\MongoDB\Server\5.0.1

### PR DESCRIPTION
**Pull request by Trenly**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131310)

### CODE REVIEW

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Change:

```
License: Server Side Public License (SSPL)
LicenseUrl: https://www.mongodb.com/licensing/server-side-public-license
```

Reference: https://www.mongodb.com/licensing/server-side-public-license/faq

**Comment by Trenly** [comment]
No time, but still working on it

File: manifests/m/MongoDB/Server/5.0.1/MongoDB.Server.locale.en-US.yaml [hunk 1]
```diff
@@ -11,8 +11,8 @@ PrivacyUrl: https://www.mongodb.com/legal/privacy-policy
 Author: MongoDB Inc.
 PackageName: MongoDB Server
 PackageUrl: https://www.mongodb.com/try/download/community
-License: Proprietary
-LicenseUrl: https://www.mongodb.com/community/licensing
+License: Server Side Public License (SSPL)
+LicenseUrl: https://www.mongodb.com/licensing/server-side-public-license
 Copyright: © 2021 MongoDB, Inc.
 CopyrightUrl: https://www.mongodb.com/community/licensing
```
Comment (mdanish-kh) [inline manifests/m/MongoDB/Server/5.0.1/MongoDB.Server.locale.en-US.yaml:17]:
```suggestion
CopyrightUrl: https://www.mongodb.com/licensing/server-side-public-license
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Not sure how I became blind to this URL, but good thing I can request changes here now

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**